app/helpers/application_helper.rb

```diff
@@ -44,4 +44,16 @@ def flash_class(level)
   def og_tag(type, options = {})
     tag.meta(property: "og:#{type}", **options)
   end
+
+  def resource_name
```

Comment:
for devise ?

Reply:
yes, for devise. The devise view calls some internal methods like resource and resource_name. So we need to map these to use them in our controllers and specs